docs/upgrade.rst

```diff
@@ -4,7 +4,7 @@ Upgrade
 Version 2.3.2
 -------------
 
-* Support for printing in a cloud environment and in an environment without sticky sessions : generates a requested printout metadata json file and stores it in the tempDir. With this information peristed, there is no need for sticky sessions and printing module can be used in a cloud environment with shared volume.
+* Support for printing in a cloud environment and in an environment without sticky sessions : generates a requested printout metadata json file and stores it in the tempDir. With this information persisted, there is no need for sticky sessions and printing module can be used in a cloud environment with shared volume.
```

Comment:
So how does this work in a could environment - don’t they need to share a temp folder or something?

Reply:
Hello @jodygarnett , 
I elaborated my documentation a bit more. 
Answering your question: depending on the cloud environment, engineers an pick an option to share a volume between multiple instances and pass the path in the existing MAPFISH_PDF_FOLDER environment variable.
I have already updated installation.rst file with the documentation for MAPFISH_PDF_FOLDER.

I think the steps for creating a shared volume is not related with this PR.

Reply:
Hi @jodygarnett, i think everything is done from our side.
Do you need more information?